zerver/webhooks/slack/doc.md

```diff
@@ -0,0 +1,61 @@
+Get notifications from Slack for messages on your team's public channels!
+
+There are two ways in which you may want to receive a notification:
+
+- Slack Channel to a single Zulip stream with different topics
+- Multiple Zulip streams for multiple Slack channels
+
+If you want a single Zulip stream to receive notifications from Slack then
+first create your desired stream and subscribe all interested parties to
+this stream. We recommend the name `{{ integration_name }}`.
```

Comment:
@aero31aero: Still we should be using those macros whereever possible for conciseness and consistency. I highly recommend it. :)